dags/map_reproducibility/utils.py

```diff
@@ -56,6 +60,7 @@ def clone_gob():
       "reproducible-benchmark-recipes",
       "cd reproducible-benchmark-recipes/projects",
       "cd gpu-recipes",
+      "pwd",
```

Comment:
I assume `pwd` will be used in followup PR?
(if not, seems not really needed here, you can delete it; if it's used in followup PR, ignore this comment)

Reply:
Yes, will be used for setting sys path